teleop_twist_keyboard.py

```diff
@@ -135,7 +135,20 @@ def main():
     rclpy.init()
 
     node = rclpy.create_node('teleop_twist_keyboard')
-    pub = node.create_publisher(geometry_msgs.msg.Twist, 'cmd_vel', 10)
+
+    # parameters
+    node.declare_parameter('stamped', False)
+    node.declare_parameter('frame_id', '')
+
+    stamped = node.get_parameter('stamped').value
+    frame_id = node.get_parameter('frame_id').value
```

Comment:
I think you should be able to shorten this, and also add in some error checking:

```suggestion
    stamped = node.declare_parameter('stamped', False)
    frame_id = node.declare_parameter('frame_id', '')
    if not stamped and frame_id:
        raise Exception("'frame_id' can only be set when 'stamped' is True")
```